index.html

```diff
@@ -2153,18 +2153,9 @@ <h2 id="OfflineAudioContext">
               <li>
                 <p>
                   Let <dfn>[[rendered buffer]]</dfn> be an internal slot
-                  initialized with an <a>AudioBuffer</a> constructed by
-                </p>
-                <pre>
-new AudioBuffer({
-    numberOfChannels: contextOptions.numberOfChannels,
-    length: contextOptions.length,
-    sampleRate: contextOptions.sampleRate
-})
-                </pre>
-                <p>
-                  This buffer holds the rendered audio data for the offline
-                  context.
+                  initialized to <code>undefined</code>. When rendering begins,
+                  this slot will hold an <a>AudioBuffer</a> containing rendered
+                  audio data for the offline context.
```

Comment:
Do we really need this at all?  The construction of the context doesn't need to say anything because you don't need one until `startRendering()` is called.

Reply:
Yeah, I don't think we need to say anything about [[rendered buffer]] anymore. So, remove all of step 4.

But this would require updating `startRendering` to define the slot.